packages/apt_deb.go

```diff
@@ -50,6 +62,24 @@ var (
 	dpkgErr = []byte("dpkg --configure -a")
 )
 
+func formatDpkgFieldsMappingToFormatingString(fieldsMapping map[string]string) string {
+	fieldsDescriptors := make([]string, 0, len(fieldsMapping))
+
+	for name, selector := range fieldsMapping {
+		//format field name and it's selector to one single entry separated by ":" and each of them wrapped in quotes
+		//name:source_name, selector:${source:Package -> ""source_name":"${source:Package}"".
+		fieldsDescriptors = append(fieldsDescriptors, fmt.Sprintf("\"%s\":\"%s\"", name, selector))
+	}
+
+	//sort descriptors to get predictable result.
+	sort.Strings(fieldsDescriptors)
+
+	//return string to format all information in json
+	//Example: {"package":"${Package}","architecture":"${Architecture}","version":"${Version}","status":"${db:Status-Status}"...}\n
+	//see dpkgInfoFieldsMapping for full set of fields.
+	return "{" + strings.Join(fieldsDescriptors, ",") + "}\n"
```

Comment:
This will construct a JSON with trailing coma at the end. Just want to double check that we are ok with that (as supportability depends on json version).

Reply:
See https://go.dev/play/p/86sLL5bkKJi